### PR TITLE
New Webstorm Cheat Sheet

### PR DESCRIPTION
**Pull request by universeofsolutions**
###### Description of changes

My IA Page: [IA Page](https://duck.co/ia/view/webstorm_mac_cheat_sheet)
A Cheat Sheet for [Webstorm](https://www.jetbrains.com/webstorm/) Shortcuts.
The Shortcuts are for Mac Users.

-

Instant Answer Page: https://duck.co/ia/view/webstorm_mac_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @universeofsolutions

### CODE REVIEW

**Comment by daxtheduck** [comment]
@universeofsolutions Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png)

**Comment by mintsoft** [comment]
@universeofsolutions Thanks for the PR; I think we should add OSX or Mac into the cheatsheet name so we can keep the mac/linux/windows PRs separate

File: share/goodie/cheat_sheets/json/webstorm.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "id": "webstorm_cheat_sheet",
+    "name": "WebStorm Cheat Sheet for Mac OS",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/webstorm.json]: WebStorm for Mac OS 
would be enough as name

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png?VMYM3KOpga)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png?n6YuzFQ6qp)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=WebStorm Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/webstorm_cheat_sheet_mobile.png?KNhyzbGPIT)

**Pull request closed without merging**